setup.py

```diff
@@ -1,14 +1,24 @@
 from setuptools import setup, find_packages
+import os
+
+
+def get_version(rel_path):
+    here = os.path.abspath(os.path.dirname(__file__))
+    with open(os.path.join(here, rel_path), 'r') as f:
+        for line in f.readlines():
+            if line.startswith('__version__'):
+                delim = '"' if '"' in line else "'"
+                return line.split(delim)[1]
+        else:
+            raise RuntimeError('Unable to find version string.')
 
-from flake8_use_fstring import __version__
 
 extra_test = [
     'coverage==4.*',
     'pytest>=4',
     'pytest-cov>=2',
 
     'flake8-builtins',
-    'flake8-commas',
```

Comment:
[`flake8-commas`](https://github.com/PyCQA/flake8-commas) is no longer installable on python 3.12 because distutils is removed. The project is longer maintained. Had to drop it to test on python 3.12.

Reply:
I came up with a possible solution: Not installing it on 3.12 only. `'flake8-commas ; python_version < "3.12"',`